triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -434,7 +461,8 @@ class Runner(
     } yield (acsResponses.flatMap(x => x.activeContracts), offset)
   }
 
-  // Run the trigger given the state of the ACS.
+  // Run the trigger given the state of the ACS. The msgFlow argument
```

Comment:
`msgFlow` isn’t a kill switch in general, e.g., in the tests we set it to something like `take(n)` to cut off the stream after n messages.